driver-core/src/main/com/mongodb/annotations/Beta.java

```diff
@@ -47,6 +47,25 @@
         ElementType.PACKAGE,
         ElementType.TYPE })
 @Documented
-@Beta
+@Beta(Beta.Reason.CLIENT)
 public @interface Beta {
+    /**
+     * @return The reason an API element is marked with {@link Beta}.
+     */
+    Reason[] value();
```

Comment:
I don't think that having a default value here is a good idea, because it has a good chance of causing us to just use `@Beta` in the future forgetting that we need to think about the reason. And each time we see just `@Beta` we would be guessing "is it really `CLIENT`, or did we forget to specify `SERVER`?".

An additional, but a much lesser point (just a technicality): some of the existing APIs are actually in `CLIENT` beta, like `AwsCredential`, while others, like `WindowedComputation` are in `SERVER` beta. Well, they were marked as beta because of the server, but we need to get rid of those annotations. So we can't technically rely on the default value until we remove `@Beta` from the API related to window functions.

Reply:
I suppose explicit over implicit is better its just kind of ugly.